src/components/maps.js

```diff
@@ -7,7 +7,7 @@ const Maps = () => {
         width="100%"
         height="600"
         id="gmap_canvas"
-        src="https://maps.google.com/maps?q=jadibuti,kathmandu&t=&z=15&ie=UTF8&iwloc=&output=embed"
+        src="https://maps.google.com/maps?q=bansbari,kathmandu&t=&z=15&ie=UTF8&iwloc=&output=embed"
```

Comment:
Lets use environment variables instead of hardcode links, a through .env file.
@saphalbhattarai Merging it for now, creating new ticket to add env variables